nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1379,7 +1490,10 @@ async fn project_snapshots_get_snapshot(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-/// Delete a snapshot from a project.
+/** Delete a snapshot
```

Comment:
There are a bunch of places like this where the summary line follows directly after the start of the block comment. I'm guessing you didn't mean to do that?

Reply:
Def did not good eyes will fix